### DISCUSSION TITLE
Rofi showing games I no longer have

### DISCUSSION THREAD

**Discussion by Kshitij-Tardalkar** [davatorium/rofi#1559] [Q&A]
I recently installed a few games on my Linux machine. 
![image](https://user-images.githubusercontent.com/50481437/147719061-9e3915a9-06b2-45d4-a697-27098cf17c88.png)
The problem is that even after I uninstalled all of them, I still see them in my Rofi menu. How do I manually remove them?

**Comment by TonCherAmi** [comment]
If you're using the drun mode, this is likely caused by some leftover `.desktop` files. The usual places to look would be `$HOME/.local/share/applications`, `/usr/share/applications`, and `/usr/local/share/applications`.

**Reply by DaveDavenport** [reply]
If you run:

```bash
G_MESSAGES_DEBUG=Dialogs.DRun rofi -show drun
```

it shows every file it parses, this might help tracking the desktop file down.

**Reply by Kshitij-Tardalkar** [reply]
Thanks a bunch. Problem is solved
